### DISCUSSION TITLE
Move the list of blocked users ON BLOCKCHAIN

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#154] [Medium CIPs]
This update should be simple. When I block someone, their Public Key must be added to the list of users I block and this must be on the blockchain. Why? Well, because there will be 100s nodes, and there is no point to block someone on one node and they still have access to your account via some other nodes.

@maebeam @diamondhands0

**Comment by tijno** [comment]
@lazynina worked on it 4 months ago

https://github.com/bitclout/core/commit/e055661b773defae4c1d3b204d0b29a78091fb13

Not sure why this was not finished/rolled out.

**Comment by arhebbar** [comment]
This makes sense! Agree with the need for this, 100%. I have blocked 100s of people for various reasons (mostly SPAM and SCAM) and it is disturbing to know that they still have access to my content. 

Having said that, isn't this something that can become a new node that allows users to see content of those who you've blocked (Unblockclout.com).

**Comment by brootle** [comment]
@maebeam @diamondhands0 @bluepartyhat @redpartyhat any update on this issue? just tell me that this is on the table, thanks!😉

**Comment by maebeam** [comment]
This is a great idea and we would love to move this information on chain ASAP
